lib/matplotlib/backend_bases.py

```diff
@@ -1164,7 +1165,7 @@ def get_sketch_params(self):
         """
         return self._sketch
 
-    def set_sketch_params(self, scale=None, length=None, randomness=None):
+    def set_sketch_params(self, scale=None, length=128, randomness=16):
```

Comment:
On a bit more configuration, this breaks passing in `None` for `length` and `randomness` (as this falls through (eventually) to `Py::Float(sketch_params[0]);` with no extra validation.  This should stay consistent with the convention of 'passing `None` means do the default'.